bridge/server/.jest/test.utils.ts

```diff
@@ -40,10 +41,16 @@ export class TestUtils {
   }
 
   public static async setupOAuthTest(): Promise<Express> {
-    process.env.OAUTH_ENABLED = 'true';
-    process.env.OAUTH_CLIENT_ID = 'myClientID';
-    process.env.OAUTH_BASE_URL = 'http://localhost';
-    process.env.OAUTH_DISCOVERY = 'http://localhost/.well-known/openid-configuration';
-    return init();
+    return init(this.config);
   }
+
+  public static readonly config = getConfiguration({
```

Comment:
Could you rename it to `OAuthConfig` or something that makes clear that this is used for OAuth?